_posts/2024-01-25-WASI-Preview2.md

```diff
@@ -0,0 +1,206 @@
+---
+title: WASI Preview 2 Launched
+author: "Dan Gohman"
+github_name: sunfishcode
+---
+> Original post from https://blog.sunfishcode.online/wasi-preview2/
+
+The WASI Subgroup has just voted to launch WASI Preview 2! This blog post
```

Comment:
```suggestion
The WASI Subgroup has just voted to launch WASI 0.2! This blog post
```

Reply:
Can we put the "formerly known as 'Preview 2'" clarification here as the opening sentence in the post and the first opportunity to do so (rather than as the next change below)?

Reply:
Sure.